draft-ietf-ipsecme-rfc8229bis.xml

```diff
@@ -1067,12 +1067,37 @@
         </list>
         <xref target="RFC5961" /> discusses how TCP injection attacks can be mitigated.
         </t>
-        <t>
-        Note, that data injection attacks are also possible on IP level (e.g. when IP fragmentation is used)
+
+        <t>Note, that data injection attacks are also possible on IP level (e.g. when IP fragmentation is used)
         resulting in DoS attack even if TCP encapsulation is not used. On the other hand, TCP injection attacks are easier to mount 
         than the IP fragmentation injection attacks, because TCP keeps a long receive window open that’s a sitting target for such attacks. 
         </t>
 
+        <t>If an attacker successfully mounts an injection attack on a TCP connection encapsulating IPsec traffic, 
+        and a Length field in the TCP stream is changed as a result of this attack, then the receiver 
+        in most cases will not be able to correctly identify the boundaries of the following packets in the stream,
+        because it will try to parse an arbitrary data (the result of encryption) as ESP (or IKE) header.
+        This will fail and for this reason all the following packets will be dropped. Eventually the situation should self-recover, but it may 
+        take several minutes and may result in IKE SA deletion and re-creation.
+        </t>
```

Comment:
I suggest no to go into details and say:

s/Communication will recover once a new connection is made/Communication will eventually recover 

otherwise we'll be asked what connection (TCP or IPsec) and since there may be many possibilities (either TCP eventually sends an RST, or IKE timeout occurs), I think we can omit all the details here - the main point that it will took quite a long.

Reply:
That's fine with me, yeah.